### PR TITLE
Add module5

### PR DESCRIPTION
**Pull request by ins21**

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -17,8 +20,20 @@ const startServer = () => {
   const app = express();
 
   app.use(express.json());
+  app.use(infoLogger);
   app.use('/api', usersRouter, suggestUsersRouter, groupsRouter, usersGroupRouter);
+  app.use(errorLogger);
   app.listen(process.env.PORT || 3000, () => console.log('Server is running'));
+
+  process.on('uncaughtException', err => {
```
Comment (Vladlan) [inline app/app.js:28]: If this error happens, server also should return res with 500.

File: app/middlewares/logger/errorLogger.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-unused-vars */
+import { logger } from './logger';
+import { getLogText } from './utils';
+
+export const errorLogger = (err, req, res, next) => {
+  const logText = getLogText(req);
+
+  logger.error(`Error: ${err.message}. Happened with this request: ${logText}`);
```
Comment (Vladlan) [inline app/middlewares/logger/errorLogger.js]: Read task 5.2  - middleware should log all unhandled errors (but you only logging error messages).

Comment (Vladlan) [reply app/middlewares/logger/errorLogger.js]: Logging error without a call stack is useless

**Review by Vladlan — CHANGES_REQUESTED** [review]

File: app/middlewares/logger/errorLogger.js (hunk 2 above)
Comment (ins21) [reply app/middlewares/logger/errorLogger.js]: Fixed it, thanks

File: app/app.js (hunk 1 above)
Comment (ins21) [reply app/app.js:28]: As we discussed, i will leave only logging for this.

**Review by Vladlan — APPROVED** [review]

**Pull request merged**